net-dns/knot/knot-2.0.2.ebuild

```diff
@@ -88,6 +88,10 @@ src_install() {
 	fi
 
 	newinitd "${FILESDIR}/knot.init" knot
+
+	if use systemd; then
```

Comment:
Just mend your mistake, commit and push again to see the change come through on Github.